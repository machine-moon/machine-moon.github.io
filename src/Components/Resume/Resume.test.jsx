import { test, expect } from "@jest/globals";
import { render } from "@testing-library/react";
import Resume from "./Resume";

test("renders Resume component correctly", () => {
  const { asFragment } = render(<Resume />);
  expect(asFragment()).toMatchSnapshot();
});
