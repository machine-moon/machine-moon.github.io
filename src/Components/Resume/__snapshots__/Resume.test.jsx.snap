// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Resume component correctly 1`] = `
<DocumentFragment>
  <div>
    Resume
  </div>
</DocumentFragment>
`;
